composer.json

```
@@ -19,17 +19,17 @@
         }
     ],
     "require": {
-        "php": "~7.1",
+        "php": "^7.2.5",
         "hanneskod/classtools": "~1.0",
-        "illuminate/support": "5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/support": "^7.5",
```

```suggestion
        "illuminate/contracts": "^7.5",
        "illuminate/support": "^7.5",
```

There might be other Illuminate packages that are depended upon. Let's make them explicit.